services/net/content/appsettings.json

```diff
@@ -52,7 +52,8 @@
       "GroupId": "Content",
       "BootstrapServers": "kafka-broker-0.kafka-headless:9092,kafka-broker-1.kafka-headless:9092,kafka-broker-2.kafka-headless:9092",
       "AutoOffsetReset": "Earliest",
-      "MaxInFlight": 5
+      "MaxInFlight": 5,
+      "EnableAutoCommit": false
```

Comment:
We need to manually commit handled messages so that we ensure we don't miss any due to errors.